tests/test_models.py

```diff
@@ -1084,3 +1137,206 @@ def test_lots_of_sweeps(self):
             random_seed=2,
         )
         assert all(tree.num_roots == 1 for tree in ts.trees())
+
+
+class TestNeutralFixation:
+    """
+    Tests for the neutral fixation model.
+    """
+
+    def test_model_end_broken(self):
+        # Checking that we're correctly detecting the fact that
+        # sweeps are non renentrant.
+        model = msprime.NeutralFixation(
+            position=0.5, start_frequency=0.1, end_frequency=0.9, dt=0.01
+        )
+        with pytest.raises(RuntimeError, match="does not support interruption"):
+            msprime.sim_ancestry(10, model=model, end_time=0.0001)
+
+    def test_incorrect_num_labels(self):
+        model = msprime.NeutralFixation(
+            position=0.5, start_frequency=0.1, end_frequency=0.9, dt=0.01
+        )
+        for num_labels in [1, 3, 10]:
+            # Not the best error, but this shouldn't be exposed to the user anyway.
```

Comment:
This test is also unclear--`num_labels` is an undocumented part of the public API.  What's going on here?

Reply:
right-- `num_labels` refers to the number of labels we use internally in the structured coalescent. we generalized the bookkeeping of nodes to include a `label` a ways back. here we are making sure that the model returns the correct number of labels, in this case 2